bool ur_activate_safeguard
bool ur_disengage_safeguard
bool ur_disengage_protective
